### ISSUE TITLE
"Not Allowed" response from Kraken API

### ISSUE THREAD

**Issue by dewey** [aaronk6/Kraken-MoneyMoney#15]
Hey, I'm seeing the following error coming from Kraken. I can see it both with version 1.07 and 1.10 (Which I've just installed). The API key has the correct permissions I believe and the [Kraken status page](https://status.kraken.com) doesn't report any issues.

![Screenshot 2024-01-31 at 19 06 06 png@2x](https://github.com/aaronk6/Kraken-MoneyMoney/assets/790262/f839475b-ab28-4075-97db-1fb801676be6)

```
The API server of your bank responded with the error message:
»Not Allowed«

Please try again later or contact the support of your bank with this error message.
```

Anyone else running into that?

**Comment by cubistico** [comment]
Seeing the same. Perhaps only a server hiccup?

**Comment by aaronk6** [comment]
I can reproduce it. My code is doing POST requests on all API routes, which worked in the past, but apparently stopped working today. However, when I call `https://api.kraken.com/0/public/AssetPairs` using GET, it still works. GET actually makes more sense here since this request doesn’t modify anything on the server side, so maybe it was never supposed to work like this and now got fixed on the API side. I’ll do some more testing and will then release an update.

**Comment by aaronk6** [comment]
Fixed with b9b2537.

**Comment by dewey** [comment]
@aaronk6 Thank you so much for this super fast fix!
